web/components/add-funds-modal.tsx

```diff
@@ -0,0 +1,56 @@
+import { manaToUSD } from 'common/util/format'
+import { useState } from 'react'
+import { useUser } from 'web/hooks/use-user'
+import { checkoutURL } from 'web/lib/service/stripe'
+import { Button } from './button'
+import { Modal } from './layout/modal'
+import { FundsSelector } from './yes-no-selector'
+
+export function AddFundsModal(props: {
+  open: boolean
+  setOpen(open: boolean): void
+}) {
+  const { open, setOpen } = props
+
+  const user = useUser()
+  // in hook so not run in nextjs
+  const [location] = useState(window.location.href)
```

Comment:
This still could get run on the server. Only a function within a `useEffect` would not be run, but the main render method is executed as normal javascript.